cli/cortx-prvsnr-cli.spec

```diff
@@ -54,27 +80,38 @@ rm -rf %{buildroot}
 # TODO test
 # TODO IMPROVE current workaround is to prevent conflicts
 #              with provisioner main rpm instllation
-cp -fpr /opt/seagate/cortx/provisioner/cli/src/* /opt/seagate/cortx/provisioner/cli
-chmod -R 750 /opt/seagate/cortx/provisioner/cli
+/bin/cp -fpr /opt/seagate/cortx/provisioner/cli/src/* /opt/seagate/cortx/provisioner/cli
+/bin/chmod -R 750 /opt/seagate/cortx/provisioner/cli
 
 # TODO test
-mkdir -p /root/.ssh
+/bin/mkdir -p /root/.ssh
 
 # Ensure update replaces the keys
 if [[ -e /root/.ssh/id_rsa_prvsnr ]]; then
-  rm -f /root/.ssh/id_rsa_prvsnr
-  rm -f /root/.ssh/id_rsa_prvsnr.pub || true
+  /bin/rm -f /root/.ssh/id_rsa_prvsnr || true
+  /bin/rm -f /root/.ssh/id_rsa_prvsnr.pub || true
+fi
+
+/bin/cp -pr /opt/seagate/cortx/provisioner/files/.ssh/id_rsa_prvsnr /root/.ssh/
+/bin/cp -pr /opt/seagate/cortx/provisioner/files/.ssh/id_rsa_prvsnr.pub /root/.ssh/
+/bin/cat /root/.ssh/id_rsa_prvsnr.pub >>/root/.ssh/authorized_keys
+
+
+if [[ ! -e /root/.ssh/ssh_config ]]; then
+  /bin/cp -pr /opt/seagate/cortx/provisioner/files/.ssh/ssh_config /root/.ssh/
 fi
 
-cp -pr /opt/seagate/cortx/provisioner/files/.ssh/* /root/.ssh/
-cat /root/.ssh/id_rsa_prvsnr.pub >>/root/.ssh/authorized_keys
-chmod 700 /root/.ssh/
-chmod 600 /root/.ssh/*
+/bin/chmod 700 /root/.ssh/
+/bin/chmod 644 /root/.ssh/*
```

Comment:
config file should have permissions 600. this line will cause ssh to faile with "bad permission or ownership of the file" error. 
config file should be configured separately in this section

Reply:
Yes.. you are right. i will update it. Thanks
```     
~/.ssh/config
             This is the per-user configuration file.  The file format and
             configuration options are described in ssh_config(5).  Because of
             the potential for abuse, this file must have strict permissions:
             read/write for the user, and not writable by others.
```
http://linuxcommand.org/lc3_man_pages/ssh1.html